src/actions/__tests__/importText.ts

```diff
@@ -1453,3 +1453,37 @@ it('import plaintext with embedded <li> mixed with text', () => {
       - y
     - d`)
 })
+
+it('import markdown', () => {
```

Comment:
This test looks correct, but when paste the input text into the UI, it does not detect that it is markdown and imports it like this:

```
- # H1
- a
- ## H2
- b
- ### H3
- c
- #### H4
- d
- ##### H5
- e
- ###### H6
- f
```

Reply:
This was caused by `useOnPaste` using `importFiles` when detecting line breaks. Fixed by adding a markdown check and prefering `importText` in that case.